frontend/src/utils/chart/TreeMap.js

```diff
@@ -45,6 +50,10 @@ const TreeMap = (data, extra, selected) => {
             },
           },
         },
+        itemStyle: {
+          color: "#FFB800",
```

Comment:
Could we move this style constants to a centralised place?

Reply:
For map style, I think we can centralize it after we have a bunch of charts loaded because this style is different than we use on CSS.
Or do we need to do it now?

Reply:
moving it to `chart-style.js`